openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -446,7 +447,7 @@ def get_email_existence_validation_error(email):
     :return: Validation error message.
 
     """
-    return _validate(_validate_email_doesnt_exist, errors.AccountEmailAlreadyExists, email)
+    return _validate(validate_email_exists_or_retired, errors.AccountEmailAlreadyExists, email)
```

Comment:
let's rename this function as we are validating that the email should NOT already exist and should NOT be retired already. So maybe something like,`validate_email_not_retired_nor_exists`?